site/contents/write_ups/2021/03-11-21_writeup.md

```diff
@@ -0,0 +1,109 @@
+
+(As a quick note, ombudsman is used a gender neutral term, as the suffix "-man" is a gender neutral suffix in the Swedish language. Regardless, the term has fallen out of usage in recent years.)
+
+How will the UK approach this problem across different sectors? Would there be a national/ethical committee for every industry? It sounds to us like a lot of hoops to jump through before we reach a solution.
+
+We saw issue with a seeming dispraity between public and private sector regulation. Whilst the report outlined the public sector restrictions, it didn't seem like there would be any regulations in the private sector which is troubling but frankly not surprising.
+
+What do the authors mean by innovation? Some of us feared a barrage of "new technologies" which will simply be a collection of linear regression soups garnished with a few "for" and "if" loops?
+
+## How do you see regulation as potentially impacting your work? If you are not UK based, are there similarities/differences with any of your national regulations?
+
+Is the European approach better? Does it give us more confidence? They have guidelines from the EU.
+ 
+As we mentioned before with bureaucracy, the ethical processes can be really messy when lots of people get involved. What makes things trustworthy to who? Who should be involved in these decision makers? Is it better that the public are more soothed by the decision (populist, religious figures for example) or if there is a stronger technical and ethical base for the decision making? How do we decide what makes a strong "ethical base"? 
+
+Should we seek to reassure the public with the decision making? We discussed the importance of diversity in decision making; different research backgrounds, different races, genders, sexuality etc. The more different experiences people have, the more likely we are to find an issue. We decided that being more likely to find ethical issues is a good thing, at the very least for ethical guidelines, if not necersarily for innovation. Of course if we wanted everyone to be happy we would get never get anything done (as noted last time in our discussion on Decolonising Academia). Even the best decision makers might not be trusted by the public. But are the general public the best people to decide what being trustworthy means?
+
+Having different use-cases for different sectors allows flexibility in decision making. Rigidity in guidelines is important for drawing lines we should refuse to cross but needs to be adaptive as our technological capabilities and societal standards shift.
+
+It was good that this report acknowledged fairness bias, accountability and the commission on race and ethnic disparity. However, it wasn't clear the report was sure on the practicalities of what they wanted - do they want new laws and legislation? 
```

Comment:
```suggestion
It was good that this report acknowledged fairness, bias, accountability and the commission on race and ethnic disparity. However, it wasn't clear the report was sure on the practicalities of what they wanted and there was some frustration amongst us that often all reports call for is more reports, rather than making solid recommendations. That said, it is also important to be sure that every issue is being give the time and space it needs to be considered fully.
```